src/components/PhotoFrame.tsx

```diff
@@ -241,6 +242,28 @@ const PhotoFrame = ({
         }
     }, [open]);
 
+    useEffect(() => {
+        const livePhotoVideo: HTMLVideoElement =
+            document.querySelector('video');
```

Comment:
there can be multiple video elements na . Does this ensure that we get the selector the current opened one ?